### ISSUE TITLE
Use RAAML for threat models

### ISSUE THREAD

**Issue by kuwv** [OWASP/threat-dragon#639]
**Describe what problem your feature request solves**

The Risk Analysis and Assessment Modeling Language (RAAML) specification is a sysml compliant that would allow integration with other modeling capabilities such as simulation.

**Describe the solution you'd like**

Implement modeling capabilities to support RAAML either as default or as conversion.

**Additional context**

https://www.omg.org/spec/RAAML/1.0/Beta2/About-RAAML

**Comment by jgadsden** [comment]
This would be a good feature for either version 2.2 or maybe later

**Comment by kuwv** [comment]
Seems like another OWASP project is doing ontology-driven threat modeling. I wonder if any of that could be reusable here.

@nets4geeks

**Comment by nets4geeks** [comment]
It is easy to make RDF from OWL, and I used to parse and modify Threat Dragon json configs in order to add advanced threats, however I think this does not help.

I did not find any mention regarding RDF and even DFD in RAAML specification.
As I understand RAAML is based on SysML and does not deal with OWL, RDF, etc.

**Comment by kuwv** [comment]
Yeah, RAAML is SysML based. To state it more accurately MBSE can use SysML with OSLC to provide interoperability between other components through RDF.

https://www.omgwiki.org/OMGSysML/lib/exe/fetch.php?media=sysml-oslc:mapping_sysml_to_rdf_vocabulary_-_april_28_2014.pdf

**Comment by jgadsden** [comment]
no implementation of this so far, so bumping this to version 2.4
